lmdeploy/turbomind/__init__.py

```diff
@@ -1,5 +1,4 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-from .tokenizer import Tokenizer
 from .turbomind import TurboMind
 
 __all__ = ['Tokenizer', 'TurboMind']
```

Comment:
may remove 'Tokenizer'

```suggestion
__all__ = ['TurboMind']
```

Reply:
done